ui/tests/e2e/connections.spec.js

```diff
@@ -151,6 +151,35 @@ test(
   },
 );
 
+test('Delete Kubernetes cluster connections', { tag: '@unstable' }, async ({ page }) => {
+  // Navigate to 'Connections' tab
+  await page.getByRole('tab', { name: 'Connections' }).click();
+  // Find the row with the connection to be deleted
+  const row = page.locator('tr').filter({ hasText: 'connected' }).first();
+
+  //find the checkbox in the row
+  const checkbox = row.locator('input[type="checkbox"]').first();
+  await checkbox.check();
+
+  // Click "Delete" button in the table
+  await page.getByRole('button', { name: 'Delete', exact: true }).click();
+  // Verify that Confirmation modal opened and delete
+  await expect(page.getByText('Delete Connections')).toBeVisible();
+
+  await page.getByRole('button', { name: 'Delete', exact: true }).click();
+
+  const statusReq = await page.waitForResponse(
+    (response) =>
+      response
+        .url()
+        .startsWith(`${ENV.MESHERY_SERVER_URL}/api/integrations/connections/meshery/status`) &&
+      response.status() === 202,
+  );
+
+  // Verify that the status of the connection is deleted
+  expect(statusReq.ok()).toBe(false);
```

Comment:
once the api returns status of ok we can safely assume the row was deleted.

Reply:
to be false or [await expect(response).toBeOK()](https://playwright.dev/docs/api/class-apiresponseassertions#api-response-assertions-to-be-ok)?

Reply:
good catch. i was testing both happy/unhappy paths and forgot to flip it back before committing. 😄